Gopkg.toml

```diff
@@ -58,8 +58,8 @@ required = [
   version = "1.6.1"
 
 [[constraint]]
+  branch = "master"
```

Comment:
This should rather be locked with `revision`, just like govpp, otherwise it will get updated on each `make dep-install` (dep ensure).